Sources/RInAppMessaging/RInAppMessaging.swift

```diff
@@ -60,11 +60,14 @@ import RSDKUtils
     /// - Parameters:
     ///     - subscriptionKey: your app's subscription key. (This setting will override the `InAppMessagingAppSubscriptionID` value in Info.plist)
     ///     - configurationURL: a configuration URL. (This setting will override the `InAppMessagingConfigurationURL` value in Info.plist)
-    ///     - enableTooltipFeature: set to `true` to enable Tooltip campaigns. This feature is currently in beta phase. Default value: `false`
+    ///     - enableTooltipFeature: set to `true` to enable Tooltip campaigns. This feature is currently in beta phase. Default value: `false`.
+    ///     - caller: for internal use only.
     @objc public static func configure(subscriptionID: String? = nil,
                                        configurationURL: String? = nil,
-                                       enableTooltipFeature: Bool = false) {
-        guard initializedModule == nil else {
+                                       enableTooltipFeature: Bool = false,
+                                       caller: String = #file) {
```

Comment:
It is not a breaking change, but is it okay to update public API?

Reply:
@imaginaris Can we follow the same approach followed by Android?

Reply:
@maureenorea-clores I couldn't find any other way to identify if the `configure()` is called by RMC or not.
I think it's ok as long as we don't break the API.

Reply:
@maureenorea-clores Or actually we have just decided to use the same approach for RMC identification used in Android 😬 . So the public API will not be modified

Reply:
@imaginaris I see, okay thanks